src/Spatial/Internals/AvlTreeSet/AvlNode.cs

```diff
@@ -0,0 +1,80 @@
+﻿namespace MathNet.Spatial.Internals
+{
+    public sealed class AvlNode<T>
```

Comment:
make it internal if it is not meant to be used from the outside

Reply:
heh missed one, ok will fix